include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -140,12 +144,11 @@ typedef enum OrtErrorCode {
 
 // The actual types defined have an Ort prefix
 ORT_RUNTIME_CLASS(Env);
-ORT_RUNTIME_CLASS(Status);  // nullptr for Status* indicates success
```

Comment:
I'd suggest to keep the comments (which is good).

Reply:
But for the CreateStatus function, it's not true.

Reply:
That seems like an edge case though (and hopefully obvious to the caller), so the comment is helpful for every other usage. If you're concerned about OrtCreateStatus maybe note in the comment here or add documentation to OrtCreatetStatus saying it returns a non-nullptr for success.

---
In reply to: [314542648](https://github.com/microsoft/onnxruntime/pull/1609#discussion_r314542648) [](ancestors = 314542648)